### ISSUE TITLE
npm WARN deprecated uuid@3.4.0

### ISSUE THREAD

**Issue by bdbvb** [MyPureCloud/genesys-cloud-webrtc-sdk#817]
When doing a fresh install of genesys-cloud-webrtc-sdk, I get some warnings about the uuid package:

```
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
```

This is appears to be an outdated package used by the following three packages:

- genesys-cloud-client-logger
- genesys-cloud-webrtc-sdk
- softphone-vendor-headsets

**Comment by hjon** [comment]
@bdbvb Thanks for raising this!

Sorry for the churn on closing/opening; it's been a while since I've worked with a GitHub ticket across repos and using commit messages to resolve them. That should all clear up once we get the SDK updated.

**Comment by bdbvb** [comment]
@hjon @ININDevEvangelists 
I still see this issue:

![image](https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/assets/11095196/cd1f30b8-8d93-49a5-ba7a-83fa6ab8dd3b)

**Comment by hjon** [comment]
@bdbvb Thanks for pointing that out. Looks like I lost track of it. Opened a PR: https://github.com/MyPureCloud/genesys-cloud-webrtc-sdk/pull/850

**Comment by hjon** [comment]
This should finally be resolved with v9.0.7.
